### ISSUE TITLE
template-hello-world-ts    does not pull latest :(

### ISSUE THREAD

**Issue by NathanWalker** [NativeScript/nativescript-cli#2045]
https://github.com/NativeScript/template-hello-world-ts
^ I cannot post issues there so I'll post here.

I assumed that running `tns create app-name --tsc` (as mentioned in the README on that template) would pull the latest from that template but appears it pull some old version :(
`tns create my-app-name --template tns-template-hello-world-ts` does not pull latest either.

I want to use this shorthand at UtahJS. Would be great for it to use latest.

/cc @tjvantoll @dtopuzov @vakrilov

**Comment by tjvantoll** [comment]
@dtopuzov Could you publish a 2.2.1 tag for that template to npm so we can get the latest changes?

**Comment by tjvantoll** [comment]
And just to note for others, until that happens you can always pull from master using:

```
tns create app-name-2 --template https://github.com/NativeScript/template-hello-world-ts`
```

**Comment by vchimev** [comment]
Hi, Guys!

By design, the `tns create` command looks for the latest version of the `js/ts/ng` template published to npm as it has been tested and released successfully.

As @tjvantoll suggested, you can use the source of the template from the `master` branch as follows:
```
tns create app-name --template https://github.com/NativeScript/template-hello-world-ts.git#master
```

The next version of this template which will be published to *npm* is 2.3.0.

I pull requested an update of the README of the template [here] (https://github.com/NativeScript/template-hello-world-ts/pull/13).
